components/Ui/UiJoinWithCards.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <section class="cds--grid join-with-cards">
+    <div class="cds--row">
+      <article
+        class="cds--col-lg-4 cds--col-md-4 join-with-cards__feature join-with-cards__intro"
+      >
+        <slot name="description" />
+        <UiCta
+          v-if="cta"
+          is-wider
+          :label="cta.label"
+          :segment="cta.segment"
+          :url="cta.url"
+        />
+      </article>
+      <div
+        v-for="card in cards"
+        :key="card.name"
+        class="cds--col-lg-4 cds--col-md-4 join-with-cards__feature"
+      >
+        <UiCompactFeature
+          :description="card.description"
+          :icon="card.icon"
+          :title="card.name"
+        />
+      </div>
+    </div>
+  </section>
+</template>
+
+<script setup lang="ts">
+import { TextLink } from "~/types/links";
+
+interface Card {
+  name: string;
+  description: string;
+  icon: string;
+}
+
+interface Props {
+  cta?: TextLink;
+  cards: Card[];
+}
+
+defineProps<Props>();
+</script>
+
+<style lang="scss" scoped>
+@use "~/assets/scss/carbon.scss";
+
+.join-with-cards {
+  margin-bottom: carbon.$spacing-12;
```

Comment:
I think that the component itself should not include the definition of the outer margin.

The component that uses it (**pages/advocates.vue** and **pages/ecosystem.vue**) is the one that is aware of how much space there needs to be around this component.

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3443/commits/83cbb3b3030bb36070a9ad5860b1b8d55b12cfe7